source/docs/guides/apple_pay/index.md

```diff
@@ -39,6 +39,20 @@ To enable Apple Pay on your Merchant Account, you'll set a password for your P12
 3. Ensure your File Format is set to **.Personal Information Exchange (.p12)** before clicking **Save**.
 4. Enter the password you'll use when uploading your .p12 to the Member Area and click **OK**.
 
+>VERY IMPORTANT NOTE: You must upload the ***"ApplePay Payment Processing Certificate".***
+For furter information visit: [Configure Merchant ID and Certificates](https://developer.apple.com/documentation/apple_pay_on_the_web/configuring_your_environment)
+
+>If you use a different certificate instead ***"ApplePay Payment Processing Certificate"*** you will could get the error: 
```

Comment:
Wording needs some fixes here. Should it say "instead of ***"?
Also the end of this sentence needs a fix. "you will could get the error" can change to "you may get the error".

Reply:
Done!